odl/solvers/functional/default_functionals.py

```diff
@@ -1467,14 +1470,15 @@ def __repr__(self):
 
 
 class KullbackLeiblerCrossEntropyConvexConj(Functional):
-    """The convex conjugate of Kullback-Leibler Cross Entorpy functional.
+
+    r"""The convex conjugate of Kullback-Leibler Cross Entorpy functional.
```

Comment:
Convex conjugate of the Kullback-Leibler Cross Entorpy functional.

Reply:
And fix the typo "Entorpy"